test/e2e/clusterpropagationpolicy_test.go

```diff
@@ -1057,6 +1057,7 @@ var _ = ginkgo.Describe("[Suspend] clusterPropagation testing", func() {
 		ginkgo.DeferCleanup(func() {
 			framework.RemoveClusterPropagationPolicy(karmadaClient, policy.Name)
 			framework.RemoveClusterRole(kubeClient, clusterRole.Name)
+			framework.WaitClusterRoleBindingDisappearOnCluster(targetMember, clusterRole.Name)
```

Comment:
ClusterRoleBinding -> ClusterRole ?

Reply:
During the cleanup, maybe we do not need to wait for resources to be cleared. The content to be checked should have been overwritten by other E2E test cases.